tests/test/ambiguity_issue_727.rs

```diff
@@ -0,0 +1,133 @@
+use super::*;
```

Comment:
I feel like I'd like to see these tests more properly categorized. But I don't know how strongly I feel about it.

Reply:
Makes sense-- but I'm not sure where to categorize them!